code/modules/jobs/job_types/_job.dm

```diff
@@ -169,6 +169,9 @@
 
 	return max(0, minimal_player_age - C.player_age)
 
+/datum/job/proc/get_title(var/mob/living/H)
```

Comment:
```suggestion
/datum/job/proc/get_title(mob/living/H)
```

Reply:
it's really up to preference

Reply:
Was pretty sure this was in the style guide, oh well.

Reply:
Not having `var` in it is preferable, yes.